daemon/containerd/image_list.go

```diff
@@ -32,9 +34,16 @@ var acceptedImageFilterTags = map[string]bool{
 	"until":     true,
 }
 
+// byCreated is a temporary type used to sort a list of images by creation
+// time.
+type byCreated []*types.ImageSummary
+
+func (r byCreated) Len() int           { return len(r) }
+func (r byCreated) Swap(i, j int)      { r[i], r[j] = r[j], r[i] }
+func (r byCreated) Less(i, j int) bool { return r[i].Created < r[j].Created }
```

Comment:
Not a blocker, but starting to wonder if we need to move this somewhere central (I was planning to move the image types from `types` to `types/image`, so perhaps that's a good opportunity).

Also considering that sorting may at some point be something we want on the client side as well